src/scenes/Landing/MoveSummary.jsx

```diff
@@ -510,3 +528,5 @@ export const MoveSummary = props => {
     </Fragment>
   );
 };
+
+export const MoveSummary = props => withContext(<MoveSummaryWithoutContext {...props} />);
```

Comment:
I think this can be simplified to:

```suggestion
export const MoveSummary = withContext(MoveSummaryWithoutContext);
```